import "IpcMessage.proto";

package METRO.CRO.MESSAGES;

message ResponseLogin {
	extend IpcMessage {
		optional ResponseLogin message = 701;
	}

	optional string username = 1;
	optional string status = 2;
}